README.md

```diff
@@ -102,7 +103,7 @@ resources:
 Sample request after deploying.
 
 ```bash
-curl -X POST https://xxxxxx.execute-api.us-east-1.amazonaws.com/dev/sqs -d '{"message": "testtest"}' -H 'Content-Type:application/json'
+curl https://xxxxxx.execute-api.us-east-1.amazonaws.com/dev/sqs -d '{"message": "testtest"}' -H 'Content-Type:application/json'
```

Comment:
Why removing? `curl` default behavior is GET, right?
Those examples are specified `post` in `method` param.

Reply:
I found out about it recently:
https://daniel.haxx.se/blog/2015/09/11/unnecessary-use-of-curl-x/

If you specify `-d` curl will send a `POST request.

`-X` should be used when you actually want to override the method

Reply:
ok, I didn't know that. thank you for letting me know 😄

Reply:
I've read that post several times, been using `curl` without `-X` for a few weeks now and still can't get my head around it :smiley: